spec/tictactoe_spec.rb

```diff
@@ -7,7 +7,25 @@
   let (:test_ttt) { TicTacToe.new(game_io, player1, player2) }
 
   it "runs the game loop " do
-    allow(test_ttt.game_loop).to receive(:game_loop) { }
-    test_ttt.game_loop
+    expect(test_ttt.game_loop).to receive(:game_loop).once
+    expect(game_io).to receive(:puts_message)
+    expect(game_io).to receive(:get_input).and_return("N")
+    test_ttt.play_game
+  end
+
+  it "plays the game twice " do
+    expect(test_ttt.game_loop).to receive(:game_loop).twice
```

Comment:
I didn't change the name of the test. It had a game_loop in my `TicTacToe` class before, but I made a separate `GameLoop` class. The method I'm testing in `TicTacToe` is the `play_move` method, which does make a call to the `GameLoop` class. I should have changed the code to `it "test play_game" do`